bitsandbytes/functional.py

```diff
@@ -1013,7 +1027,10 @@ def dequantize_blockwise(
     return out
 
 
-def get_4bit_type(typename, device=None, blocksize=64):
+DataType4Bit = Literal["nf4", "fp4", "int4", "af4"]
+
+
+def get_4bit_type(typename: DataType4Bit, device: Optional[Union[int, str, torch.device]] = None, blocksize=64):
```

Comment:
Trailing comma, type for `blocksize`?

Reply:
`blocksize` should be inferred as `int` here.